help/C/Help_ch_Transactions.xml

```diff
@@ -891,6 +891,12 @@ Author:
           </para></listitem>
           <listitem><para><emphasis role="bold"><link linkend="trans-import-csv">CSV</link></emphasis> 
                (.csv)  Comma Separated Values;</para></listitem>
+          <listitem><para><emphasis role="bold"><link linkend="trans-import-MT940-MT942-DTAUS">MT940</link></emphasis> 
+               (.STA)  SWIFT MT940 Customer Statement Message;</para></listitem>
+          <listitem><para><emphasis role="bold"><link linkend="trans-import-MT940-MT942-DTAUS">MT942</link></emphasis> 
+               (.txt  SWIFT MT942 Interim Transaction Report;</para></listitem>
+          <listitem><para><emphasis role="bold"><link linkend="trans-import-MT940-MT942-DTAUS">DTAUS</link></emphasis> 
+               (.DTAUS)  Banking data Exchange File;</para></listitem>
```

Comment:
The name is derived from german "Datenträgeraustauschverfahren". [https://de.wikipedia.org/wiki/Datentr%C3%A4geraustauschverfahren]. First used 1976 with 9-track magnetic tapes, later other media like floppy. Bank support ended 1. Februar 2016.

I am not  sure: should we add this infos in the chapter or the glossary?